lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -50,8 +50,8 @@ def annotate_plugin(env, status, response_headers, response_body)
     def trace!(span, zipkin_env, &block)
       # if the request comes from a non zipkin-enabled source record the default tags
       tags = DEFAULT_SERVER_RECV_TAGS unless zipkin_env.called_with_zipkin_headers?
-      # if the user specified tags to record on server recv, use these no matter what
-      tags = @config.record_on_server_receive if @config.record_on_server_receive
+      # if the user specified tags to record on server receive, use these no matter what
```

Comment:
? Am I missing something? Any spelling issue with the following?
"if the user specified tags to record on server receive, use these no matter what"